src/vs/workbench/api/common/extHostQuickOpen.ts

```diff
@@ -510,8 +517,8 @@ export function createExtHostQuickOpen(mainContext: IMainContext, workspace: IEx
 		return typeof iconPath === 'object' && 'dark' in iconPath ? iconPath.dark : iconPath;
 	}
 
-	function getIconPathOrClass(button: QuickInputButton) {
-		const iconPathOrIconClass = getIconUris(button.iconPath);
+	function getIconPathOrClass(icon: QuickInputButton['iconPath']) {
```

Comment:
I wonder if there's a utility function somewhere that does this exact same code. I feel like we use icons all over the place in the API that there has to be some reusability here.